components/Modal/Profile/ProfileModal.tsx

```diff
@@ -19,50 +19,33 @@ import {
   Text,
 } from "@chakra-ui/react";
 import { useRouter } from "next/router";
-import React, { useRef, useState } from "react";
+import React, { useRef, useState, useEffect } from "react";
 import { MdAccountCircle } from "react-icons/md";
 import { useSemaphoreAuthState } from "@/hooks/useSemaphoreAuthState"; 
 
+
 type ProfileModalProps = {
   open: boolean;
   handleClose: () => void;
 };
 
 const ProfileModal: React.FC<ProfileModalProps> = ({ open, handleClose }) => {
-  const [user, userLoading, userError, , ] = useSemaphoreAuthState();
-  const { selectedFile, setSelectedFile, onSelectFile } = useSelectFile(300, 300);
-  const selectFileRef = useRef<HTMLInputElement>(null); // Correctly type the ref
+  const [user, userLoading, userError] = useSemaphoreAuthState();
```

Comment:
Can you double check this to see if it's working properly (if it isn't)? I added a couple functions to the `useSemaphoreAuthState()` export and thought it would require something like `[user, userLoading, userError, , ]` to account for the other values.

Reply:
My bad, I’m sure it’s.